index.html

```diff
@@ -1332,6 +1300,33 @@
           </li>
         </ol>
       </li>
+      <li id="adapter-inbox">
+        Let <var>receivedMessage</var> be associated with <var>adapter</var>,
```

Comment:
We use ["internal slots"](http://ecma-international.org/ecma-262/6.0/#sec-object-internal-methods-and-internal-slots) to refer to implementation-accessible attributes of Javascript objects. Streams and Web Bluetooth are [using the `adapter@[[slot]]` convention](https://webbluetoothcg.github.io/web-bluetooth/#terminology) to refer to these.